codegen/src/parse.rs

```diff
@@ -437,7 +437,7 @@ mod parsing {
                 let content;
                 braced!(content in input);
                 input.parse::<Token![;]>()?;
-                content.parse::<token::Dollar>()?;
+                content.parse::<Token![$]]>()?;
```

Comment:
This line does not compile because the delimiters are not balanced.
https://travis-ci.org/dtolnay/syn/jobs/550313981

Reply:
Ah I didn't realize that `cargo test --release --all-features` isn't enough, otherwise I'd have noticed this before opening the PR.